templates/taxi/car_list.html

```diff
@@ -1,17 +1,18 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-  <h1>Car list</h1>
-  {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
-  {% else %}
-    <p>There are no cars in taxi</p>
-  {% endif %}
+  <h1>
+    {% if object %}
+      Update Car
+    {% else %}
+      Create Car
+    {% endif %}
```

Comment:
The logic for determining whether to display 'Update Car' or 'Create Car' is based on the presence of `object`. Ensure this logic accurately reflects the intended context, possibly by checking if the form is bound or by using a more explicit condition.